tests/test_searcher.py

```diff
@@ -0,0 +1,79 @@
+from mock import patch
+import unittest
+
+from search.searcher import Searcher
+
+
+class SearcherTests(unittest.TestCase):
+
+    # Argument values
+    REPO_NAME = 'haiti'
+    MAX_RESULTS = 10
+    # Return values
+    EXTERNAL_SEARCH_RETURN_VALUE = ['external search return value']
+    FULLTEXT_RETURN_VALUE = ['full-text return value']
+    INDEXING_RETURN_VALUE = ['indexing return value']
+
+    def setUp(self):
+        pass
+
+    @patch('external_search.search')
+    def test_external_search_backend_results(self, external_search_mock):
+        """Return external search backend results when available."""
+        external_search_mock.return_value = (
+            SearcherTests.EXTERNAL_SEARCH_RETURN_VALUE)
+        external_search_value = 'abc external search'
```

Comment:
Should be `external_search_backends` or `external_search_backends_value`. Also I believe it is a list, not a string. It may not matter here though.

Reply:
Done. What the value is here doesn't really matter, I just need to confirm that it passes the value through to the external_search.search function.

Reply:
That's true. I just wondered if it can still confuse readers whether it takes a string or a list. Up to you though.